R/EngineGraphQLGitHub.R

```diff
@@ -116,20 +116,32 @@ EngineGraphQLGitHub <- R6::R6Class("EngineGraphQLGitHub",
     }
   ),
   private = list(
+    # @field token A token authorizing access to API.
```

Comment:
Should it be commented or it was to be a documentation? If the latter then **'** character is missing.

Reply:
Roxygen does not support well documenting R6 private methods - it pops out a lot of warnings when you `devtools::document()` - there was an issue created for that on `roxygen` github site but with no reasonable workout (just closed, one of the solutions proposed is to make documentation as simple comments - I think the way it is done in Python) https://github.com/r-lib/roxygen2/issues/1016